requestfinder.py

```diff
@@ -1,29 +1,25 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
-from ConfigParser import ConfigParser
+from configparser import ConfigParser
 from xdg.BaseDirectory import load_first_config
 from xml.etree import cElementTree as ET
```

Comment:
if is python3 only "his module will use a fast implementation whenever available. The xml.etree.cElementTree module is deprecated." from python 3.3

Reply:
It still exists there for backward compatible, that's why you can found cElementTree everywhere  in this repo, maybe at some point we should change it by another PR.